lib/datasource/index.ts

```diff
@@ -196,6 +197,9 @@ async function fetchReleases(
       });
     }
   } catch (err) {
+    if (err.messages === HOST_DISABLED || err.err?.message === HOST_DISABLED) {
```

Comment:
this looks strange `err.messages` -> `err.message` ? And why `err.err` ?

Reply:
Because it gets wrapped in an ExternalHostError sometimes